### ISSUE TITLE
closing tag exclusion can be bypassed by entering from multiple cursors

### ISSUE THREAD

**Issue by revelt** [bradleyflood/auto-id-class#14]
You can't add a single new class or id on a closing tag, but you can if you somehow manage to enter via multiple cursor, starting from legit opening tag:

![multiple-cursors](https://user-images.githubusercontent.com/8344688/38070169-698fdf02-3312-11e8-84d8-f74759e61221.gif)

This is sneaky-one but I imagine fix should be possible. If we could check _each_ cursor's surroundings somehow...

**Comment by bradfx** [comment]
Yep I assume we need to run the checks for every cursor instead of just the one, as per @alexfabianoricioli 's work:
```
    for (var i = 0; i < atom.workspace.getActiveTextEditor().cursors.length; i++) {
      atom.workspace.getActiveTextEditor().cursors[i].moveLeft(1);
    }
```
etc.

**Comment by revelt** [comment]
hmm... Very intriguing... I exceeded my today's free time allowance, need to move onto client work but I will look asap

If somebody picks this up no hurt feelings 🥇 otherwise I'll try

**Comment by bradfx** [comment]
Haha Yep I know exactly what you mean 💯

**Comment by ricioli** [comment]
I thought of two ways to solve this problem, in the two I will analyze each cursor separately, the difference is:
1. In one I will insert the point in the ones in the closing and the attribute in the others, one at a time. This would result in undoing a ctrl + z for each cursor. With this option in the future it could even check if a class attribute already exists and in this tag just move the cursor within the class quotes, but it can also cause the error if the programmer has a cursor outside the visible code area in the editor (Ex : </div.your-class-name>).
2. Another is to remove the cursors that are in invalid positions (as in closing the tag) and then insert the attribute at one time in the others.

What is your opinion?

Ps: I'm Brazilian so sorry for my bad english.
